index.html

```diff
@@ -26,23 +27,38 @@
     </section>
     <nav  id="desktopNav">
       <ul>
-        <li class="navTab" data-content="home"><a href="index.html" class="icon-home">Home</a></li>
-        <li class="navTab" data-content="aboutMe"><a href="aboutMe.html" class="icon-about">About Me</a></li>
-        <li class="navTab" data-content="projects"><a href="projects.html" class="icon-projects">Projects</a></li>
+        <li class="navTab" data-home="home" id="homeNav">Home</a></li>
+        <li class="navTab" data-aboutMe="aboutMe" id="aboutMeNav">About Me</a></li>
+        <li class="navTab" data-projects="projects" id="projectsNav">Projects</a></li>
```

Comment:
I would consider making these anchor tags (within the list item) because it's hard to tell that the list items are clickable from a user interface perspective.